source/usage-and-configuration/swagger/swagger-ui.md

```diff
@@ -13,8 +13,14 @@ To include [swagger ui](https://swagger.io/swagger-ui/) javascript interface in
 title: Petstore Api Reference
 ---
 
-{% swagger_ui ./petstore.yaml %}
+{% swagger_ui <swagger path> %}
 ```
+>The location of swagger file (`<swagger_path>`) can be a local file path or the URL of a file reachable on internet.
+Examples:
```

Comment:
The double list seems to much here....

Why not 

#### Examples

With a local path:

```md
{% swagger_ui ./petstore.yaml %}
```

With a URL

```
{% swagger_ui  https://raw.githubusercontent.com/OAI/OpenAPI-Specification/master/examples/v2.0/json/petstore.json %}
```